src/main/java/bank/account/AccountDTO.java

```diff
@@ -0,0 +1,9 @@
+package bank.account;
+
+public record AccountDTO(
+        String id,
+        String iban,
+        int balance,
+        Long personId
```

Comment:
```suggestion
        String personId
```
має бути Person.getUid() - зовнішній ідентифікатор
не треба віддавати ідентифікатор з бази даних